erizo/src/erizo/MediaStream.cpp

```diff
@@ -447,7 +451,7 @@ void MediaStream::initializePipeline() {
   pipeline_->addService(packet_buffer_);
 
   pipeline_->addFront(std::make_shared<PacketReader>(this));
-
+  addHandlerInPosition(BEGINNING, handler_pointer_dic, handler_order);
```

Comment:
I like these three options. An alternative naming would be TOP, MIDDLE, and BOTTOM, which might give a more accurate description of where the handler will be placed in the "network" stack.